### DISCUSSION TITLE
Does this have auto update?

### DISCUSSION THREAD

**Discussion by jwkim930** [LizardByte/community#117] [Sunshine] [answered]
Couldn’t find this anywhere in the documentation or other discussions. Does it have a built-in updater (or at least a notification that a newer version is available), or do I have to check for new releases manually?

**Comment by ReenigneArcher** [comment, marked as answer]
Not currently, it's in our roadmap though. For now we have notifications on all of our social platforms listed here (except YouTube). https://app.lizardbyte.dev/?lng=en#Social

Plus you can watch the repo for releases.

**Comment by KINGLIFER** [comment]
How do you know what version you are running???

**Reply by ReenigneArcher** [reply]
`sunshine --version`

**Reply by KatyPurry** [reply]
> `sunshine --version`

Sorry if it's a stupid question, I have sunshine on Windows, where do I need to put this command ?
Thank you
